docs/resource-cleanup.md

```diff
@@ -67,14 +67,14 @@ a. remove rbd image(csi-vol-omapval, the prefix csi-vol is value of [volumeNameP
   Removing image: 100% complete...done.
   ```
 
-b. remove cephfs subvolume(csi-vol-omapval)
+b. remove cephFS subvolume(csi-vol-omapval)
 
   ```
   ceph fs subvolume rm volume_name subvolume_name group_name
   ```
 
   ```bash
-  ceph fs subvolume rm  cephfs csi-vol-340daf84-5e8f-11ea-8560-6e87b41d7a6e csi
+  ceph fs subvolume rm  cephFS csi-vol-340daf84-5e8f-11ea-8560-6e87b41d7a6e csi
```

Comment:
this should be cephfs

Reply:
ok. I thought since its a volume name we can modify it.

Reply:
Yes  it can be but looks odd :)